go.mod

```diff
@@ -149,3 +149,5 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 	tags.cncf.io/container-device-interface/specs-go v0.8.0 // indirect
 )
+
+replace github.com/containerd/containerd/api => ./api
```

Comment:
Should the API changes be merged in a separate PR first, so that the `go.mod` can be updated to use a pseudo-version / commit (and prevent a replace rule)?

Reply:
No, better to do them all together. The tagging just needs to occur before the next release and can be done as part of the release process

Reply:
But in that case, anyone using the containerd module must also add a `replace` rule to get the correct API version with these changes, wheras merging the API changes first allows the `go.mod` to be bumped to a version `>` current release.